qiita_db/util.py

```diff
@@ -863,7 +863,7 @@ def convert_to_id(value, table, text_col=None):
                                      % (value, table))
         # If there was a result it was a single row and and single value,
         # hence access to [0][0]
-        return _id[0][0]
+        return int(_id[0][0])
```

Comment:
I'm curious about this change... why is it needed? which is the specific case that was failing? the id in the database should already be an integer, so psycopg2 should return an int.